docs/src/contributing.md

```diff
@@ -99,3 +119,43 @@ Finally, click "Create pull request".
 You may get some questions about it, and possibly suggestions of how to make it ready to go into the main project. 
 If you had test errors or problems, we are happy to help you. 
 Then, if all goes according to plan, it gets merged... **Thanks for the contribution!!** :tada: :tada: :tada:
+
+## Note on Adding Dependencies
+
```

Comment:
Maybe mention that we try to not have too many dependencies? 
I can't remember if it's mentioned somewhere so if not: The developer should create an issue first before working on a PR.